test/Service.test.js

```diff
@@ -77,7 +77,7 @@ describe('Testing Service class', function() {
       this.timeout(config.testTimeout2);
 
       service.taskInfoList().then((taskInfoList) => {
-        expect(taskInfoList).to.be.an.array; 
+        expect(taskInfoList).to.be.an.array;
```

Comment:
Make sure this is doing what you expect. Chai's "a/an" method is a function http://chaijs.com/api/bdd/#method_a